EPRN.Portal/Views/Areas/Reprocessor/PRNS/ActionPRN.cshtml

```diff
@@ -0,0 +1,38 @@
+﻿@using EPRN.Common.Enums;
```

Comment:
This is a PERN, not a PRN - by being in Exporter area makes this a PERN. Therefore any reference to PRN in the view needs to be changed to PERN